src/backend/EdFi.DataManagementService.Backend.Postgresql/Deploy/Scripts/0001_Create_Document_Table.sql

```diff
@@ -35,3 +35,24 @@ CREATE TABLE dms.Document_15 PARTITION OF dms.Document FOR VALUES WITH (MODULUS
 
 -- GET/UPDATE/DELETE by id lookup support, DocumentUuid uniqueness validation
 CREATE UNIQUE INDEX UX_Document_DocumentUuid ON dms.Document (DocumentPartitionKey, DocumentUuid);
+
+-- Set REPLICA IDENTITY FULL to all partitions so all columns are
+-- available through replication to e.g. Debezium
+ALTER TABLE dms.Document REPLICA IDENTITY FULL;
+ALTER TABLE dms.Document_00 REPLICA IDENTITY FULL;
+ALTER TABLE dms.Document_01 REPLICA IDENTITY FULL;
+ALTER TABLE dms.Document_02 REPLICA IDENTITY FULL;
+ALTER TABLE dms.Document_03 REPLICA IDENTITY FULL;
+ALTER TABLE dms.Document_04 REPLICA IDENTITY FULL;
+ALTER TABLE dms.Document_05 REPLICA IDENTITY FULL;
+ALTER TABLE dms.Document_06 REPLICA IDENTITY FULL;
+ALTER TABLE dms.Document_07 REPLICA IDENTITY FULL;
+ALTER TABLE dms.Document_08 REPLICA IDENTITY FULL;
+ALTER TABLE dms.Document_09 REPLICA IDENTITY FULL;
+ALTER TABLE dms.Document_10 REPLICA IDENTITY FULL;
+ALTER TABLE dms.Document_11 REPLICA IDENTITY FULL;
+ALTER TABLE dms.Document_12 REPLICA IDENTITY FULL;
+ALTER TABLE dms.Document_13 REPLICA IDENTITY FULL;
+ALTER TABLE dms.Document_14 REPLICA IDENTITY FULL;
+ALTER TABLE dms.Document_15 REPLICA IDENTITY FULL;
```

Comment:
`REPLICA IDENTITY FULL` is needed for PostgreSQL to emit full row on delete, which we need to derive the correct Opensearch index.